_layouts/default.html

```diff
@@ -1,79 +1,39 @@
-            script_done = true;
-            if (jquery_done) {
-                {{page.js_callback}}
-            }
-            script.onload = script.onreadystatechange = null;
-        }
-    };
-    document.body.appendChild(script);
-
-{% endif %}
-}
-
-if (window.addEventListener)
-    window.addEventListener("load", loadjs, false);
-else if (window.attachEvent)
-    window.attachEvent("onload", loadjs);
-else window.onload = loadjs;
-        </script>
-{% endif %}
-    </head>
+<head>
+    <meta charset=utf-8">
+    <title>i3 - improved tiling wm</title>
+    <link rel="icon" href="/favicon.ico">
+    <link rel="stylesheet" href="/css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="i3 is a dynamic tiling window manager with clean, readable and documented code, featuring extended Xinerama support, usage of libxcb instead of xlib and several improvements over wmii">
+    <meta name="author" content="i3 developers">
+    {% if page.javascript != null %}
+    <script type="text/javascript" src="/js/{{page.javascript}}" defer></script>
+    {% endif %}
     <body>
-        <div id="main">
-            <a href="/"><h1 id="title">i3 - improved tiling WM</h1></a>
-			<ul id="nav">
-{% include menulink.html name="Docs" url="/docs/" %}
-{% include menulink.html name="Screens" url="/screenshots/" %}
-{% include menulink.html name="FAQ" url="https://www.reddit.com/r/i3wm/" %}
-{% include menulink.html name="Contact" url="/contact/" %}
-{% include menulink.html name="Bugs" url="https://github.com/i3/i3/issues" %}
-			</ul>
-	<br style="clear: both">
-	    {{ content }}
-        </div>
-    <div id="footer" lang="de">
-        © 2009-present Michael Stapelberg,
-        <a href="/impress.html">Impressum</a>,
-        <a href="https://github.com/i3/i3.github.io">Source</a>
-    </div>
+        <header>
+            <a class="logo" href="/">
+                <img src="/img/logo.svg" alt="i3 WM logo" />
+            </a>
+            <nav>
+                <ul>
+                    {% include menulink.html name="Docs" url="/docs/" %}
+                    {% include menulink.html name="Screens" url="/screenshots/" %}
+                    {% include menulink.html name="FAQ" url="https://www.reddit.com/r/i3wm/" %}
+                    {% include menulink.html name="Contact" url="/contact/" %}
+                    {% include menulink.html name="Bugs" url="https://github.com/i3/i3/issues" %}
+                </ul>
+            </nav>
+        </header>
+        <main>
+	        {{ content }}
+        </main>
+        <footer>
+            © 2009-present Michael Stapelberg,
+            <a href="/impress.html">Impressum</a>,
+            <a href="https://github.com/i3/i3.github.io">Source</a>
+        </footer>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
Can you make this inclusion of jquery (if it’s still required?) conditional on there being any javascript on the page please? It currently shows up on the index page, for example.

Reply:
62335369104fd0c7681971aa496b74fa1b238b8f